util/commit.sh.tpl

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+
+%{load_statement}
+
+output_tag=$(echo '%{original_image}:%{command}' | md5)
+
+id=$(docker run -d --privileged %{flags} %{image} %{command})
+
+docker commit $id run_and_commit:$output_tag
```

Comment:
This is missing save command which will save the image tar.
https://github.com/GoogleCloudPlatform/base-images-docker/blob/master/package_managers/install_pkgs.bzl#L38

Other rules can then consume this output by predicting the path.

Reply:
good catch, added